docs/src/index.md

```diff
@@ -82,8 +82,44 @@ There are several core parts to the Catlab design, we start with a brief overvie
 
 4. [`@present`](@ref) enumerates a finite set of generators for a model of the GAT just like you would write out a group (model of the theory of groups) as list of generators and relations, the presentation lets you enumerate the objects and morphisms that generate a category.
 
+## Conventions
 
+In several places in Catlab, we use what we call "Abstract Field Convention". Instead of doing the following:
```

Comment:
On second thought, I am wondering what is the motivation for this "abstract field convention." Usually my preference is to wrap fields with generic accessor functions, which allows for a uniform interface to different structs, and the Julia style guide suggests that one should [prefer exported methods over direct field access](https://docs.julialang.org/en/v1/manual/style-guide/#Prefer-exported-methods-over-direct-field-access). Is performance what you're worried about here?